src/Components/App/App.css

```diff
@@ -0,0 +1,7 @@
+.app {
+  width: 100vw;
+}
+
+html {
+  width: 100vw;
+}
```

Comment:
We might not need this snippet if the App class is set to 100vw

Reply:
Ya I was having CSS struggles